src/Components/test/E2ETest/Tests/WebAssemblyAuthenticationTests.cs

```diff
@@ -221,6 +221,88 @@ public void AuthenticatedUser_CanGoToProfile()
         Browser.Equal("/", () => new Uri(Browser.Url).PathAndQuery);
     }
 
+    [Fact]
+    public void CanPassAdditionalParameters_DuringSignIn()
+    {
+        // Register first user
+        ClickAndNavigate(By.PartialLinkText("Register"), "/Identity/Account/Register");
+
+        var userName1 = $"{Guid.NewGuid()}@example.com";
+        var password1 = $"[PLACEHOLDER]-1a";
```

Comment:
What is this `[PLACEHOLDER]` string for? I don't see any other code in this PR that would care what the value of the password is.

Reply:
I see some of the existing tests also use this string, but I still am not sure if it's meant to have some meaning or if it's just a completely arbitrary string (including the square brackets).

Reply:
I think this is because we generate a Guid for the password and replace it, as we need to ensure it complies with the password policy of having uppercase, lowercase, numbers and special characters, but to be honest, that is a guess. I just copy pasted an existing test to use as a starting point for these tests